### ISSUE TITLE
Query Builder as a dendepency

### ISSUE THREAD

**Issue by ezsper** [MCProHosting/cassandra-orm#9]
Please consider making query builder a separate module that can be used as a dependency for cassandra-orm. In my use case, I'm not using modeling, I'm just using the query builder.

**Comment by connor4312** [comment]
I agree with this; I was considering doing it previously but never got around to it. I'll see what I can do :wink:

**Comment by ezsper** [comment]
That's awesome! Thanks for considering it.
